hoomd/write/table.py

```diff
@@ -328,41 +328,39 @@ def __setstate__(self, state):
 
 
 class Table(_InternalCustomWriter):
-    """A delimiter separated value file backend for a Logger.
+    """Write delimiter separated values to a stream.
 
-    This can serve as a way to output scalar simulation data to standard out.
-    However, this is useable to store simulation scalar data to a file as well.
+    Use `Table` to write scalar and string `hoomd.logging.Logger` quantities to
+    standard out or to a file.
 
-    Note:
-        This only works with scalar and string quantities. If using string
-        quantities, keep in mind that the default space delimiter will make
-        strings with spaces in them will cause read errors if attempting to read
-        the outputed data with a space delimited file reader.
+    Warning:
+        When logger quantities include strings with spaces, the default space
+        delimiter will result in files that are not machine readable.
 
-    Note:
+    Important:
         All attributes for this class are static. They cannot be set to new
         values once created.
 
     Args:
         trigger (hoomd.trigger.Trigger): The trigger to determine when to run
-            the Table back end.
+            the Table backend.
         logger (hoomd.logging.Logger): The logger to query for output. The
             'scalar' categories must be set on the logger, and the 'string'
             categories is optional.
         output (``file-like`` object , optional): A file-like object to output
```

Comment:
I think its a bit ambiguous here what we mean by a "file-like object". An example or more explanation would be good. I view this as something minor and would approve this PR without this change, but its something to keep in mind moving forward.

Reply:
We list explicitly the methods required for this interface, but I am adding examples